Signal/translations/en.lproj/Localizable.strings

```diff
@@ -643,6 +643,45 @@
 /* No comment provided by engineer. */
 "EMPTY_INBOX_TITLE" = "Squeaky Freaking Clean.";
 
+/* Text field placeholder for 'two factor auth pin' when confirming a pin. */
+"ENABLE_2FA_VIEW_CONFIRM_PIN_DEFAULT_TEXT" = "••••••";
+
+/* Indicates that user should confirm their 'two factor auth pin'. */
+"ENABLE_2FA_VIEW_CONFIRM_PIN_INSTRUCTIONS" = "Confirm your PIN.";
+
+/* Error indicating that attempt to disable 'two-factor auth' failed. */
+"ENABLE_2FA_VIEW_COULD_NOT_DISABLE_2FA" = "Could not disable two-step verification.";
```

Comment:
I know you said the copy is provisional, but I think it makes sense to use whatever copy we have from the spec as a starting point (e.g. "Registration Lock" not "two-step verification"), hopefully that'll allow us to get more focused feedback when we go over this as a group.

Reply:
Yup, I'm going to fix all of the copy in the next PR.